aea/runtime.py

```diff
@@ -81,10 +83,16 @@ def __init__(
         self._multiplexer: AsyncMultiplexer = self._get_multiplexer_instance()
         self._task_manager = TaskManager()
         self._decision_maker: Optional[DecisionMaker] = None
+        self._storage: Optional[Storage] = None
```

Comment:
Storage backend should be configurable via the AEA config, so please add as optional field in aea-config and pass down

Reply:
done